src/snowflake/connector/pandas_tools.py

```diff
@@ -381,7 +422,7 @@ def drop_object(name: str, object_type: str) -> None:
             f"COPY INTO {target_table_location} /* Python:snowflake.connector.pandas_tools.write_pandas() */ "
             f"({columns}) "
             f"FROM (SELECT {parquet_columns} FROM @{stage_location}) "
-            f"FILE_FORMAT=(TYPE=PARQUET COMPRESSION={compression_map[compression]}{' BINARY_AS_TEXT=FALSE' if auto_create_table or overwrite else ''}) "
+            f"FILE_FORMAT=(TYPE=PARQUET COMPRESSION={compression_map[compression]}{' BINARY_AS_TEXT=FALSE' if auto_create_table or overwrite else ''}{sql_use_logical_type}) "
```

Comment:
can you break this into two lines

Reply:
I broke it up even further, such that each file_format argument is on a separate line.